src/client/dfuse/dfuse.h

```diff
@@ -450,6 +454,7 @@ struct dfuse_pool {
 	ACTION(LISTXATTR)                                                                          \
 	ACTION(RENAME)                                                                             \
 	ACTION(OPEN)                                                                               \
+	ACTION(PRE_READ)                                                                           \
```

Comment:
Is this new stat used anywhere?

Reply:
No, other than display none of these are yet.

I've recently made these available via json and am working on adding checking the stats into the testing, https://github.com/daos-stack/daos/pull/13927 adds the ability for ftest to read these stats from a test.